diff --git a/target/linux/rockchip/armv8/target.mk b/target/linux/rockchip/armv8/target.mk
index 2fd1292817..dca78a2de7 100644
--- a/target/linux/rockchip/armv8/target.mk
+++ b/target/linux/rockchip/armv8/target.mk
@@ -1,6 +1,7 @@
 ARCH:=aarch64
 SUBTARGET:=armv8
 BOARDNAME:=RK33xx/RK35xx boards (64 bit)
+CPU_TYPE:=cortex-a76
 
 define Target/Description
 	Build firmware image for Rockchip RK33xx/RK35xx devices.
