diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 6baa933825..2a0542f4c5 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .74
-LINUX_KERNEL_HASH-6.12.74 = 3b56eeb1dc9a437f189ca56b823be3769994f59a4ea0895b08ec0d20acaca13e
+LINUX_VERSION-6.12 = .75
+LINUX_KERNEL_HASH-6.12.75 = f71accce24dc45d614104453899609a3d1b21104183e989d0760572aeb4b3607
