diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 427a005af7..3c95b1d36e 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .80
-LINUX_KERNEL_HASH-6.12.80 = c92591d896e79ecddbc3319136f0c2f855e832b397de7593f013ad7590a43e53
+LINUX_VERSION-6.12 = .81
+LINUX_KERNEL_HASH-6.12.81 = 412b58d42bb3dd98be9ceab190e714ff426f35810aed159f68f877b32777114d
