diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index c786e6ed44..d638b2f7bf 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .69
-LINUX_KERNEL_HASH-6.12.69 = 4b493657f218703239c4f22415f027b3644949bf2761abd18b849f0aad5f7665
+LINUX_VERSION-6.12 = .70
+LINUX_KERNEL_HASH-6.12.70 = 8e51c7cee097ab44021f2e27da81eb9ec597ee638e64d3452e9b5046bc937266
