diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index f5313cd820..42b2621682 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .84
-LINUX_KERNEL_HASH-6.12.84 = d56e463a7b1f5355794ae83a102453ad930a51f9cb46b295fcf24a717bff52dd
+LINUX_VERSION-6.12 = .85
+LINUX_KERNEL_HASH-6.12.85 = e35ac999f40a6874493d8d60f33f1150d7a89ae5841c428da82257fbcd070aed
