diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 6baa933825..2a0542f4c5 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .71
-LINUX_KERNEL_HASH-6.12.71 = 143e8bc76cc41f831b51aa5e75819bed55bed41f299d35922820f1d2d2b02600
+LINUX_VERSION-6.12 = .75
+LINUX_KERNEL_HASH-6.12.75 = c507198e5d9952222fd09b35f51bb3817c6b2c722c53796ca331b0a36a7e9a68

