diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 1be2869132..def4fb0d94 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .68
-LINUX_KERNEL_HASH-6.12.68 = d367c7504bd4da520dd01eb08125d2d0ac088bc8af4cd56d236f2074dd4225b7
+LINUX_VERSION-6.12 = .69
+LINUX_KERNEL_HASH-6.12.69 = 6e044da8f85140e4890ad6b4d48e746cf0daec1d45530e8d598f4f7fef1bcbfb
