diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index c1aed9b3dc..2384dfeca0 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .55
-LINUX_KERNEL_HASH-6.12.55 = 328f8f4608a653063a5fd82d29b17163faab2825fa419fa85b961740a342fb9f
+LINUX_VERSION-6.12 = .56
+LINUX_KERNEL_HASH-6.12.56 = da694b1288e19dc968855ecba97fca690ff723a66458efbe5d68dda34c3e715f
