diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 99bf5c5390..ee8386149c 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .79
-LINUX_KERNEL_HASH-6.12.79 = 4bfa751f33de2a5d7ecb4ff964743a027fc726a2225a76a18f92f0582aa0790b
+LINUX_VERSION-6.12 = .80
+LINUX_KERNEL_HASH-6.12.80 = ea46231a09179db924c6e64b3fffaaa08ef6656b9197e7824ad2657ea45e89fc
