diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 920ea0645e..16cbc8ea00 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .58
-LINUX_KERNEL_HASH-6.12.58 = 5f1c4c546660a6a81046fdfa6195306bad2c8d17c0d69876dc100a85ad4613ac
+LINUX_VERSION-6.12 = .59
+LINUX_KERNEL_HASH-6.12.59 = cab851acc2166e44ed83983b1ca2a69006046aa828e657d4554bc119419fee17
