diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 463c22204a..834fa9956f 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .63
-LINUX_KERNEL_HASH-6.12.63 = 9502c5ffe4b894383c97abfccf74430a84732f04ee476b9c0d87635b29df7db3
+LINUX_VERSION-6.12 = .64
+LINUX_KERNEL_HASH-6.12.64 = 7c6ef1fec9734b7659d5e083df05a6adc6e6e8f263222682fd12606d1926529a
