diff --git a/package/kernel/mac80211/patches/build/000-fix.patch b/package/kernel/mac80211/patches/build/000-fix.patch
new file mode 100644
index 0000000000..24d91110f7
--- /dev/null
+++ b/package/kernel/mac80211/patches/build/000-fix.patch
@@ -0,0 +1,13 @@
+diff --git a/backport-include/linux/hrtimer.h b/backport-include/linux/hrtimer.h
+index 3c35192..3b1ba19 100644
+--- a/backport-include/linux/hrtimer.h
++++ b/backport-include/linux/hrtimer.h
+@@ -10,7 +10,7 @@
+ 
+ #endif /* < 4.16 */
+ 
+-#if LINUX_VERSION_IS_LESS(6,13,0)
++#if LINUX_VERSION_IS_LESS(6,12,0)
+ 
+ static inline enum hrtimer_restart hrtimer_dummy_timeout(struct hrtimer *unused)
+ {
diff --git a/package/kernel/mac80211/Makefile b/package/kernel/mac80211/Makefile
index 4267aa0e36..637610cfcd 100644
--- a/package/kernel/mac80211/Makefile
+++ b/package/kernel/mac80211/Makefile
@@ -10,15 +10,15 @@ include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=mac80211
 
-PKG_VERSION:=6.18
+PKG_VERSION:=6.18.1
 PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0-only
 PKG_LICENSE_FILES:=COPYING
 
-PKG_SOURCE_URL:=https://github.com/openwrt/backports/releases/download/backports-v$(PKG_VERSION)
-PKG_HASH:=3bd5d84b1e56cf8addff0d1e4bcf06753c51feee88c4d7925fb7b67784529436
+PKG_SOURCE_URL:=https://github.com/mj22226/backports/releases/download/v6.18.1/
+PKG_HASH:=0ac463f5730e1d0b9c2bc4ec999c614e997664d582c070b0e0eb78528bfa286c
 
-PKG_SOURCE:=backports-$(PKG_VERSION).tar.zst
+PKG_SOURCE:=backports-$(PKG_VERSION).tar.xz
 PKG_BUILD_DIR:=$(KERNEL_BUILD_DIR)/$(if $(BUILD_VARIANT),$(PKG_NAME)-$(BUILD_VARIANT)/)backports-$(PKG_VERSION)
 PKG_BUILD_PARALLEL:=1
 
